cds_books/migrator/records.py

```diff
@@ -233,13 +240,20 @@ def create_record(cls, dump):
             object_type='rec',
             object_uuid=record_uuid,
         )
+        # -1 should mean the latest revision
         timestamp, json_data = dump.rest[-1]
         json_data['pid'] = provider.pid.pid_value
         record.model.json = json_data
         record.model.created = dump.created.replace(tzinfo=None)
         record.model.updated = timestamp.replace(tzinfo=None)
-        document = Document.create(record.model.json, record_uuid)
-        document.commit()
-        db.session.commit()
+        try:
+            document = Document.create(record.model.json, record_uuid)
+            document.commit()
+            db.session.commit()
+
+            return document
+        except IlsValidationError as e:
+            click.secho(e.original_exception, fg='red')
+            # TODO uncomment when data cleaner
```

Comment:
nit: there are a lot of these comments, an some of them are uncommented already.